### ISSUE TITLE
[FEATURE] FD504GW-DX-Z416 Support

### ISSUE THREAD

**Issue by xcapslock** [mkst/zte-config-utility#89]
any ideas to know how to decrypt config (size is 390 kb, so encryption can not be that hard, ?) of isp customized firmware?
UI Looks like this :
![UI](https://github.com/mkst/zte-config-utility/assets/95067680/ec942260-87c6-45a8-bedd-97856f5d38e5)
here is the config link: [config.bin](https://1drv.ms/u/s!AuPJQiqz5nsHeQOl1zOemtZ--sU?e=FPh2Nf)
Telnet is already open but doesn't accept any i/p.  
![loop telnet](https://github.com/mkst/zte-config-utility/assets/95067680/52da9eb2-340a-4093-bff1-04fe338145f7)

can someone help me to how decrypt the config?  and send me pratikdahal7@outlook.com

**Comment by kingace2056** [comment]
@xcapslock  Were you able to do anything with this router ?

**Comment by xcapslock** [comment]
@kingace2056 , yes i was able to decode the config.bin file and got telnet access successfully...  
if ur router is from vianet (old model) then here is the creds: 
user: `manu`
pwd: `dea22a72` (or some have custom, derived from router MAC)

and GUI have different level login:
userlevel1: user- `vianet`   pwd- `we@!sp`
userlevel2: user- `admin`  pwd- `admin`
userlevel3: user- `manu`  pwd- `dea22a72`
also try :  user: `cdata_user` pwd- `cdata_pass`
```
(note these are 2024 login details, idk know if it still works, i've change my ISP, so i dont know if they have changed them.. ..
or u can drop you config.bin, i will decrypt it..)
```

**Comment by abishektw** [comment]
@xcapslock Could you please decrypt mine and provide me the login creds?

It is same model with same firmware version as in the screenshot you have attached.

**Comment by xcapslock** [comment]
Bro ani github bata yo tmro drive ko link delete gara haii....
& Sorry for late reply..
________________________________

**Comment by xcapslock** [comment]
see mail, i have sent u a full decrypted xml... & @abishektw delete above drive link for your privacy....

**Comment by abishektw** [comment]
@xcapslock thank You so much
